tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -197,36 +197,6 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		}
 	})
 
-	staticProvisioningPreSetUpUtil := func(ctx context.Context) (*restclient.Config, *storagev1.StorageClass, string) {
```

Comment:
This method is also present in vsphere_volume_expansion.go 

If This is removed from csi_static_provisioning.go, it takes reference from the above file vsphere_volume_expansion.go.  Instead Move this method to util.go